exercises/word-count/example.scm

```diff
@@ -0,0 +1,45 @@
+(define-module (word-count)
+  #:use-module (srfi srfi-1)
+  #:use-module (ice-9 regex)
+  #:export (countwords))
+
+
+(define (lookup key alist)
+  (if (assoc key alist)
```

Comment:
Since `assoc-ref` returns `#f` when there's no matching entry, the `assoc` check is unnecessary and we can simply use the following.

```scheme
(define (lookup key alist)
  (assoc-ref alist key))
```